assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-footer-block/block.tsx

```diff
@@ -34,12 +31,23 @@ const PaymentMethodIconsElement = (): JSX.Element => {
 };
 
 interface Props {
+	children: JSX.Element | JSX.Element[];
 	className?: string;
 	cartButtonLabel: string;
 	checkoutButtonLabel: string;
 }
 
+const hasChildren = ( children ): boolean => {
+	return children.some( ( child ) => {
+		if ( Array.isArray( child ) ) {
+			return hasChildren( child );
+		}
+		return child !== null && ! isString( child ) && child.key !== null;
```

Comment:
I think this line can be simplified like this:
```suggestion
		return isObject( child ) && child.key !== null;
```